disco_aws_automation/disco_vpc_peerings.py

```diff
@@ -111,98 +128,88 @@ def _find_peer_vpc(self, peer_vpc_id):
         except UnboundLocalError:
             raise RuntimeError("VPC {0} is missing tags: 'Name', 'type'.".format(peer_vpc_id))
 
-    @staticmethod
-    def create_peering_connections(peering_configs):
-        """ create vpc peering configuration from the peering config dictionary"""
-        client = boto3.client('ec2')
-        for peering in peering_configs.keys():
-            vpc_map = peering_configs[peering]['vpc_map']
-            vpc_metanetwork_map = peering_configs[peering]['vpc_metanetwork_map']
-            vpc_ids = [vpc.vpc['VpcId'] for vpc in vpc_map.values()]
-
-            existing_peerings = throttled_call(
-                client.describe_vpc_peering_connections,
-                Filters=create_filters({'status-code': ['active'],
-                                        'accepter-vpc-info.vpc-id': [vpc_ids[0]],
-                                        'requester-vpc-info.vpc-id': [vpc_ids[1]]})
-            )['VpcPeeringConnections']
+    def _create_peering_connections(self, peerings):
+        """ Create peerings in AWS for the given PeeringConnection objects"""
+        for peering in peerings:
+            peering_conn = throttled_call(
+                self.client.create_vpc_peering_connection,
+                VpcId=peering.source_endpoint.vpc['VpcId'],
+                PeerVpcId=peering.target_endpoint.vpc['VpcId']
+            )['VpcPeeringConnection']
+
+            # wait for the peering connection to be ready
+            waiter = self.client.get_waiter('vpc_peering_connection_exists')
+            waiter.wait(
```

Comment:
According to the boto3 documentation, `VpcPeeringConnectionIds` takes one or more peering IDs. We could probably use that to parallelize the wait.